src/PrestaShopBundle/Resources/config/services/bundle/controller.yml

```diff
@@ -17,6 +19,8 @@ services:
             movements: "@prestashop.core.api.stock.movements_collection"
             stockRepository: "@prestashop.core.api.stock.repository"
             queryParams: "@prestashop.core.api.query_stock_params_collection"
+        tags:
```

Comment:
Is it requried to add the tag on each definition? Since it's already defined on the parent?

Reply:
Yes, service inheritance seems not to keep tags (I checked)